include/NovelRTInterop/Maths/GeoVector2F_t.h

```diff
@@ -1,6 +1,10 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
 #include "NovelRT.h"
 
+#ifdef _WIN32
+#pragma warning( disable: 4099 )
+#pragma warning( disable: 4190 )
+#endif
```

Comment:
If I'm to continue with using the `typedef struct [ClassName] [CTypeName]` method, the MSVC compiler does not like this. It also does not like us returning the struct or the C++ type in the C methods, however it appears to be okay with other compilers, hence why we only check for Win32 here. If this turns out to be wrong, or if I switch away from this method, I'll remove this.